conda_build.recipe/meta.yaml

```diff
@@ -16,6 +16,7 @@ build:
 requirements:
   build:
     - python
+    - conda
```

Comment:
With out this dep you get

```

Installed /home/tcaswell/.pe/_build/lib/python2.7/site-packages/conda_build-1.12.0_7_g8468d76-py2.7.egg
Processing dependencies for conda-build===1.12.0-7-g8468d76
Searching for conda
Traceback (most recent call last):
  File "setup.py", line 54, in <module>
    package_data={'conda_build': ['templates/*', 'cli-*.exe']},
  File "/home/tcaswell/.pe/_build/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/home/tcaswell/.pe/_build/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/home/tcaswell/.pe/_build/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/command/install.py", line 67, in run
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/command/install.py", line 117, in do_egg_install
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/command/easy_install.py", line 380, in run
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/command/easy_install.py", line 610, in easy_install
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/command/easy_install.py", line 661, in install_item
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/command/easy_install.py", line 709, in process_distribution
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/pkg_resources/__init__.py", line 824, in resolve
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/pkg_resources/__init__.py", line 1069, in best_match
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/pkg_resources/__init__.py", line 1081, in obtain
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/command/easy_install.py", line 617, in easy_install
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/package_index.py", line 594, in fetch_distribution
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/package_index.py", line 431, in find_packages
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/package_index.py", line 765, in scan_url
  File "/home/tcaswell/.pe/_build/lib/python2.7/site-packages/setuptools-15.0-py2.7.egg/setuptools/package_index.py", line 283, in process_url
RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.python.org/simple/conda/r
Command failed: /bin/bash -x -e /home/tcaswell/other_source/conda-build/conda_build.recipe/build.sh
```

It looks like the setup.py requires it so it is trying to reach out to pip to install it.